cinema_service_db_data.json

```diff
@@ -0,0 +1,550 @@
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 2,
+    "fields": {
+      "first_name": "Leonardo",
+      "last_name": "DiCaprio"
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 3,
+    "fields": {
+      "first_name": "Matt",
+      "last_name": "Damon"
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 4,
+    "fields": {
+      "first_name": "Joseph",
+      "last_name": "Gordon-Levitt"
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 5,
+    "fields": {
+      "first_name": "Elliot",
+      "last_name": "Page"
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 6,
+    "fields": {
+      "first_name": "Bruce",
+      "last_name": "Willis"
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 7,
+    "fields": {
+      "first_name": "Emily",
+      "last_name": "Blunt"
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 8,
+    "fields": {
+      "first_name": "Samuel",
+      "last_name": "Jackson"
+    }
+  },
+  {
+    "model": "cinema.actor",
+    "pk": 9,
+    "fields": {
+      "first_name": "Robin",
+      "last_name": "Wright"
+    }
+  },
+  {
+    "model": "cinema.movie",
+    "pk": 1,
+    "fields": {
+      "title": "The Departed",
+      "description": "An undercover cop and a mole in the police attempt to identify each other while infiltrating an Irishgang in South Boston.",
+      "duration": 151,
+      "genres": [
+        1,
+        2,
+        3
+      ],
+      "actors": [
+        1,
+        2,
+        3
+      ]
+    }
+  },
+  {
+    "model": "cinema.movie",
+    "pk": 2,
+    "fields": {
+      "title": "Inception",
+      "description": "A thief who steals corporate secrets through the use of dream-sharing technology is given the inverse task of planting an idea into the mind of a C.E.O.",
+      "duration": 148,
+      "genres": [
+        4,
+        5,
+        6
+      ],
+      "actors": [
+        2,
+        4,
+        5
+      ]
+    }
+  },
+  {
+    "model": "cinema.movie",
+    "pk": 3,
+    "fields": {
+      "title": "Looper",
+      "description": "In 2074, when the mob wants to get rid of someone, the target is sent into the past, where a hired gun awaits - someone like Joe - who one day learns the mob wants to 'close the loop' by sending back Joe's future self for assassination.",
+      "duration": 119,
+      "genres": [
+        2,
+        4,
+        6
+      ],
+      "actors": [
+        4,
+        6,
+        7
+      ]
+    }
+  },
+  {
+    "model": "cinema.movie",
+    "pk": 4,
+    "fields": {
+      "title": "Unbreakable",
+      "description": "A man learns something extraordinary about himself after a devastating accident.",
+      "duration": 106,
+      "genres": [
+        2,
+        6,
+        7
+      ],
+      "actors": [
+        6,
+        8,
+        9
+      ]
+    }
+  },
+  {
+    "model": "cinema.moviesession",
+    "pk": 1,
+    "fields": {
+      "show_time": "2024-10-08T13:00:00Z",
```

Comment:
The `show_time` for movie sessions is set in the future. Ensure that this aligns with your testing or production needs.